saga/sagaState.go

```diff
@@ -5,28 +5,28 @@ import (
 	"fmt"
 )
 
+type flag byte
```

Comment:
Do these need to be and'ed together? It seems like having just one of the states below would be enough to know what to do? Or am I missing something?